### PR TITLE
Split Cardiovascular support into two.

### PR DESCRIPTION
**Pull request by davidmiller**

### CODE REVIEW

File: elcid/templates/records/icu_round.html [hunk 1]
```diff
@@ -14,10 +10,22 @@
...
+<span ng-show="item.inotrope_dose">
     [[ item.inotropic_dose ]]
    <br />
 </span>
+<span ng-show="item.inotrop">
+    [[ item.inotropic_drug ]]
+   <br />
+</span>
+<span ng-show="item.vasopressor_dose">
+    [[ item.inotropic_dose ]]
```
Comment (fredkingham) [inline elcid/templates/records/icu_round.html]: I think this should be `vasopressor_dose`

File: elcid/models.py [hunk 2]
```diff
@@ -841,17 +845,17 @@ class ICURound(EpisodeSubrecord):
...
         max_length=200, blank=True, null=True, choices=VENTILATION_TYPES
     )
     fio2 = models.CharField(
         max_length=20,
         blank=True, null=True, verbose_name="FiO₂"
     )
-    inotropic = models.NullBooleanField(blank=True)
-    inotropic_drug = ForeignKeyOrFreeText(InotropicDrug)
-    inotropic_dose = models.CharField(max_length=200, blank=True, null=True)
+    inotrope         = ForeignKeyOrFreeText(InotropicDrug)
```
Comment (fredkingham) [inline elcid/models.py:855]: Do you want to rename `InotropicDrug`?

Comment (davidmiller) [reply elcid/models.py:855]: not particularly - do you care lots ?

File: elcid/templates/records/icu_round.html (hunk 1 above)
Comment (davidmiller) [reply elcid/templates/records/icu_round.html]: Sure - not even the only typo in this file.
It's never rendered, so reasonably meh.

**Pull request merged**